BraveWallet/Crypto/Stores/AccountActivityStore.swift

```diff
@@ -12,6 +12,14 @@ class AccountActivityStore: ObservableObject {
   @Published private(set) var transactions: [BraveWallet.TransactionInfo] = []
   @Published private(set) var allTokens: [BraveWallet.BlockchainToken] = []
 
+  var currencyCode: CurrencyCode = .usd {
+    didSet {
+      currencyFormatter.currencyCode = currencyCode.code
+      update()
+    }
+  }
```

Comment:
Yeah I could do that. I'd have to update `CryptoStore` observer to call `update()` on these stores instead of assigning currency code but that should work fine 👍

Reply:
[e2def6a](https://github.com/brave/brave-ios/pull/5277/commits/e2def6a306943323b46a5692d2699ee37b51429a)